### DISCUSSION TITLE
[ Bug ] The official VS Code C# extension error

### DISCUSSION THREAD

**Discussion by aleks-ivanov** [community/community#4193] [Codespaces]
If you install [the extension](https://marketplace.visualstudio.com/items?itemName=ms-dotnettools.csharp) manually inside a Codespace, it installs smoothly as normal.

When installing the extension through the `devcontainer.json`, you get the following error:

```
Installing C# dependencies...
Platform: linux, x86_64, name=ubuntu, version=20.04

Downloading package 'OmniSharp for Linux (x64)' (35903 KB).................... Done!
Validating download...
Integrity Check succeeded.
Installing package 'OmniSharp for Linux (x64)'

Downloading package '.NET Core Debugger (linux / x64)' (59244 KB).................... Done!
Validating download...
Integrity Check succeeded.
Installing package '.NET Core Debugger (linux / x64)'

Downloading package 'Razor Language Server (Linux / x64)' (60640 KB).................... Done!
Installing package 'Razor Language Server (Linux / x64)'

Finished

[ERROR]: C# Extension failed to install the debugger package.
```

The result is that the C# extension is not installed.

The error seems to appear independent of project or image.

For comparison, when opening locally the same configuration that remotely returns this error, with the `Remote - Containers` extension, the error does **not** appear and the C# extension is installed as normal.

**Comment by reeceatkinson** [comment]
The developer of the extension should be able to help. You could find their details in the Visual Studio Extensions Marketplace.
https://marketplace.visualstudio.com/items?itemName=ms-dotnettools.csharp
